taxonomies/packaging_shapes.txt

```diff
@@ -277,9 +285,111 @@ fr:Batonnet, bâtonnets
 hu:rúd, rudak
 pt:Paus, pauzinhos, varetas de madeira
 
+fr:Stick
+# different - do not merge
+
 en:label, labels
 de:Etikett, Etiketten
 fr:étiquette, étiquettes
 hu:címke, címkék
 pt:Etiqueta, etiquetas, rótulo, rótulos
 weight:en:light
+
+en:Applicator
+fr:Applicateur
+packaging_materials:en: en:plastic, en:cardboard
+
+fr:Bac
+
+en:Bag-in-box
+fr:Bag-in-box, Cubitainer
+
+en:strip, band
+fr:Bandelette
+packaging_materials:en: en:plastic, en:metal, en:carboard, en:other
+
+en:Ampoule, ampul, ampule
+fr:Ampoule
+packaging_materials:en: en:glass
+
+fr:Bidon
+
+en:Blister
+fr:Blister
+
+fr:Rouleau
+
+fr:Bobine
+
+fr:Caisse
+
+fr:Coupelle
+
+fr:Bombonne
+
+fr:Collerette
+
+fr:Muselet, Muselet et plaque
+#goes_with:en:glass bottle
+
+en:hanger
+fr:Cintre
+
+en:Wedge, corner piece
+fr:Cale, Coin, Cornière
+packaging_materials:en: en:plastic, en:cardboard, en:metal, en:other
+
+# close to bottle, but also ampoule is translated to vial by citeo
+en:Vial, flask
+fr:Flacon
+
+fr:Collier
+
+en:wrapper, wrapping
+fr:enveloppage
+packaging_materials:en: en:metal, en:paper, en:other
+
+en:envelope
+fr:enveloppe
+packaging_materials:en: en:cardboard, en:paper
+
+en:string
+fr:ficelle, ruban
+
+en:individual dose
+fr:dose invididuelle
+
+fr:Housse
+
+en:pump bottle, spray, spray bottle, applicator bottle
+fr:Flacon à pompe, Flacon à pistolet, Spray
+packaging_materials:en: en:plastic, en:metal, en:glass
+
+en:Aerosol can
+fr:Aérosol
+packaging_materials:en: en:plastic, en:metal
+
+en:Ball flask, Roll-on bottle
+fr:Flacon à bille, Roll
+packaging_materials:en: en:plastic, en:glass
```

Comment:
Adding properties like that is not a good idea if we are not sure how we are going to use them. Let's wait until we do know and until we actually use them somewhere, as otherwise it will be very hard later to figure out what to do with them, or we will have unwanted side issues.

e.g. "packaging_materials:en: en:plastic, en:glass" --> if later we implement tag rules, then that would add both plastic and glass. That's probably not wanted.

Reply:
I've added them as comments